src/AutoRest.CSharp/LowLevel/Generation/LowLevelClientWriter.cs

```diff
@@ -544,6 +549,26 @@ private void WriteProtocolMethodDocumentationWithExternalXmlDoc(LowLevelClientMe
             }
         }
 
+        private void WriteConvenienceMethodOmitReason(ConvenienceMethodOmitReason reason)
+        {
+            // TODO -- create wiki links to provide guidance here: https://github.com/Azure/autorest.csharp/issues/3624
+            switch (reason)
+            {
+                case ConvenienceMethodOmitReason.SuppressedInTypeSpec:
+                    _writer.Line($"// The convenience method of this operation is not generated because the convenience method is not turned on in typespec file.");
```

Comment:
We should consider a readonly struct which has a description, wiki link, etc and then the writer can simply write what is in the struct and no case statement is needed.

Reply:
I created a class but since we do not have those entries now, I just created a new class with a `string Message` property.
We could add other things when we have those.